### PR TITLE
Updated Russian translation in /help/* files

### PR DESCRIPTION
**Pull request by rvasin**

### CODE REVIEW

File: help/generic/po/ru.po [hunk 1]
```diff
@@ -909,43 +909,43 @@ msgstr "<code>stopmusic    </code>Останавливает музыку"
 #: ../E/cheats.txt:10
 #, no-wrap
 msgid "<code>allmission   </code>All missions are unlocked, works also in main menu"
-msgstr ""
+msgstr "<code>allmission   </code>Все миссия разблокированы, работает также в главном меню"
```
Comment (probfaber) [inline help/generic/po/ru.po:912]: needs plural: миссия->миссии

**Review by probfaber — CHANGES_REQUESTED** [review]
Needs some fixies of auto translated text.

File: help/bots/po/ru.po [hunk 2]
```diff
@@ -1063,13 +1063,13 @@ msgstr ""
...
 msgid "<code>WheeledBuilder</code>"
-msgstr ""
+msgstr "<code>WheeledBuilder</code>"
 
 #. type: \b; header
 #: ../E/botbs.txt:1
 #, no-wrap
 msgid "<button 195/> Legged Builder"
-msgstr ""
+msgstr "<button 195/> Ножной Строитель"
```
[the remark this answers is not in the document]
Comment (X64E) [reply help/bots/po/ru.po:1072]: Шагающий

File: help/bots/po/ru.po [hunk 3]
```diff
@@ -1081,7 +1081,7 @@ msgstr ""
 #: ../E/botbs.txt:3
 #, no-wrap
 msgid "Ant-legged creeping bot equipped with a neutron gun."
-msgstr ""
+msgstr "Муравьиный ползучий бот, оснащенный нейтронной пушкой."
```
[the remark this answers is not in the document]
Comment (X64E) [reply help/bots/po/ru.po:1084]: Шагающий бот с нейтронной пушкой.

**Review by MrSimbax — CHANGES_REQUESTED** [review]
See other reviews. Also, there are merge conflicts now.

**Comment by X64E** [comment]
Thanks to @rvasin for make translation to Russian. But but for some reason, the errors were not fixed.
For complete work maked in this request i open new request https://github.com/colobot/colobot-data/pull/66 with corrections in trnslation. If everyone is satisfied, this request can be closed.

**Comment by MrSimbax** [comment]
Thanks!

**Pull request closed without merging**